routes/api-routes.js

```diff
@@ -15,23 +16,27 @@ module.exports = function(app) {
     });
 
     // Retrieves all events with a date field of 'startdate' or later
-    app.get('/api/event/:startdate', (req, res) => {
+    app.get('/api/event/:startdate', AccessMiddleware.hasAdminAccess, (req, res) => {
```

Comment:
this should be available for all, it's going to be used when a user picks a particular date on the landing page